modules/javafx.base/src/test/java/test/com/sun/javafx/binding/TestLogging.java

```diff
@@ -0,0 +1,24 @@
+package test.com.sun.javafx.binding;
```

Comment:
You need to add a standard copyright header.

Reply:
done